builds/e2e/proxy/configure_proxy.sh

```diff
@@ -3,26 +3,18 @@
 set -euo pipefail
 
 user="$1"
-encoded_key="$2"
-subnet_address_prefix="$3"
+subnet_address_prefix="$2"
 
-# set up SSH private key
-echo "Creating SSH private key for user '$user'"
+echo 'Installing squid'
 
-home="$(eval echo ~$user)"
-mkdir -p "$home/.ssh"
-echo -e "$encoded_key" | base64 -d > "$home/.ssh/id_rsa"
-chown -R "$user:$user" "$home/.ssh"
-chmod 700 "$home/.ssh"
-chmod 600 "$home/.ssh/id_rsa"
+for i in `seq 3`
+do
+    sleep 5
```

Comment:
Why is `sleep 5` needed here?

Reply:
VM creation was intermittently failing, so I added the retry logic (including sleep) and the problem went away.